packages/icons/src/main/java/org/talend/icons/Type.java

```diff
@@ -0,0 +1,279 @@
+/*
```

Comment:
I m not sure there is any value to versionned this file.
It will be generated by the CI so never been updated by us.
please remove it.